### ISSUE TITLE
Accessibility issues with text fields on Windows

### ISSUE THREAD

**Issue by deathdogg** [slint-ui/slint#8732]
### Bug Description

***When inputting text into a text field created by slint, screen readers behave abnormally. I do not know what the issue itself is, but based on some of the logs from my screen reader, I can guess. Many of the accessibility properties of native windows text fields don't seem to be set. The screen reader doesn't track text input correctly, braille curssors are missing, and the screen reader reads out the new value of the field any time you type a letter into it. To reproduce, you can run the NVDA screen reader, also found on github, and use the speech viewer or braille viewer to see this in action. First, go into something like notepad and observe how text tracking should be behaving. Then launch a slint app with a text field and look at the difference in behaviors.
I realize this issue might be difficult to test for, so if there's anything I can do to help, let me know

### Reproducible Code (if applicable)

```slint

```

### Environment Details

Latest version of slint on crates.io as of this writing, windows 11, using NVDA screen reader. Language used is rust, with a .slint file for **t**-**** Slint Version:
- Platform/OS:
- Programming Language:
- Backend/Renderer:

### Product Impact

*I'm making a markdown text editor with preview for the blind, so text fields working is a must**

**Comment by tronical** [comment]
Apologies. This is - unfortunately - a known issue :-(
